overlays/custom-packages.nix

```diff
@@ -66,6 +66,23 @@
             # Only override mesonFlags if colord argument is accepted
             mesonFlags = prevAttrs.mesonFlags ++ ["-Ddeprecated-color-management-colord=false"];
           });
+      systemd =
+        (prev.systemd.override {
```

Comment:
By the way I think you can just write prev.systemd.overrideAttrs if you are not going to override any of the callPackage's inputs

Reply:
Fixed it, thanks!